### ISSUE TITLE
Redshift Schema

### ISSUE THREAD

**Issue by mansours** [hapyrus/fluent-plugin-redshift#21]
This issue pertains to documentation. The community (and myself) could use some more help understanding how to use this plugin.

I'm having trouble figuring out if I need to configure my redshift table. 

For example, in the sample Java application it submits a simple json, outputs to stdout.
**2015-04-13 11:33:01 -0400 fluentd.test.follow: {"from":"userA","to":"userB"}**

Would I just created a table with the columns `from` and `to` as TEXT? and what about json with integers etc. What about timestamps? Is there a template for that?

I think I'm receiving a "received no valid data." message because no columns are matching I believe (in create_gz_file_from_structured_data)

**Comment by gerricchaplin** [comment]
That is correct just create the table with the column names so that they match the field names in the json log and you should be fine. You probably want to use varchar in the case of your example.
Just create the columns with the data_type that it should hold and as long as your data matches you should be OK.

``` 
create table proxy_logs (
col1  varchar(40)  encode  lzo,
col2  varchar(400)  encode  lzo
)
SORTKEY (col1); 
```

**Comment by mansours** [comment]
I think I figured it out, you have to wrap your column data in a parent object property named "log", also the json payload should be a string, not an object.
{
"log": "{\"from\":\"userA\", \"to\":\"userB\"}"
}

Maybe that should be more obvious in the readme for non ruby developers? (now that I know this looking at the bottom examples in ruby ut makes more sense)
